### PR TITLE
Merge remote-tracking branch 'origin/master'

### PR DESCRIPTION
**Pull request by ckmah**
Merge remote-tracking branch 'origin/master'

### CODE REVIEW

File: app/src/main/java/ckmah/mangoreader/activity/MangaItemActivity.java [hunk 1]
```diff
@@ -100,23 +100,25 @@ private void fetchMangaDetailFromMangaEden() {
...
-        });
+                    }
+                });
     }
 
     private void loadContent() {
         ((TextView) findViewById(R.id.subtitle_author)).setText(manga.getAuthor());
         CollapsingToolbarLayout collapsingToolbarLayout = (CollapsingToolbarLayout) findViewById(R.id.manga_item_collapsing_toolbar);
         collapsingToolbarLayout.setTitle(manga.getTitle());
+//        ((TextView) findViewById(R.id.chapter_title)).setText(manga.getTitle());
```
Comment (williamxu) [inline app/src/main/java/ckmah/mangoreader/activity/MangaItemActivity.java]: remove comments

File: app/src/main/java/ckmah/mangoreader/activity/MangaViewerActivity.java [hunk 2]
```diff
@@ -165,7 +165,8 @@ public void onClick(View v) {
 
     private void displayChapter() {
         // chapter title set to manga name and chapter #
-        getSupportActionBar().setTitle(mangaTitle + " - Ch. " + getChapterTitle());
+        String title = "Ch. " + getChapterTitle() + " - " + mangaTitle;
+        ((TextView) findViewById(R.id.chapter_title)).setText(title);
```
Comment (williamxu) [inline app/src/main/java/ckmah/mangoreader/activity/MangaViewerActivity.java]: What title does this show up as? 
Like "Ch. 4 - Naruto", or something else?

File: app/src/main/res/layout/activity_manga_item.xml [hunk 3]
```diff
@@ -40,6 +40,18 @@
                 android:theme="@style/MangaItemToolbar"
                 app:layout_collapseMode="pin" />
 
+            <!--<TextView-->
```
Comment (williamxu) [inline app/src/main/res/layout/activity_manga_item.xml]: wao whats dis

**Comment by williamxu** [comment]
Address comments and merge :+1:

File: app/src/main/java/ckmah/mangoreader/activity/MangaViewerActivity.java (hunk 2 above)
Comment (ckmah) [inline app/src/main/java/ckmah/mangoreader/activity/MangaViewerActivity.java]: exactly, chapter # > important than ellipsized title

**Pull request merged**